src/Umbraco.Web/Models/ContentEditing/ContentVariation.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Runtime.Serialization;
+using Umbraco.Core.Models;
+
+namespace Umbraco.Web.Models.ContentEditing
+{
+    /// <summary>
+    /// Represents the variant info for a content item
+    /// </summary>
+    [DataContract(Name = "contentVariant", Namespace = "")]
+    public class ContentVariation
+    {
+        /// <summary>
+        /// The content name of the variant
+        /// </summary>
+        [DataMember(Name = "name")]
+        public string Name { get; set; }
+        
+        [DataMember(Name = "language")]
+        public Language Language { get; set; }
+
+        [DataMember(Name = "segment")]
+        public string Segment { get; set; }
+
+        //fixme not sure if we need these dates as metadata for displaying the variant info in the drop down?
```

Comment:
@warrenbuckley good point, this is not currently used so i'll remove for now, we can add later if it's required. All of this will change immensely when we decide to allow bulk saving all variants at once.